core/src/com/unciv/ui/MultiplayerScreen.kt

```diff
@@ -431,6 +444,66 @@ class EditMultiplayerGameInfoScreen(game: GameInfo, gameName: String, backScreen
             }
         }
     }
+
+    /**
+     * Helper function to decrease indentation
+     * Turns the current playerCiv into an AI civ and uploads the game afterwards.
+     */
+    private fun giveUp(gameId: String, gameName: String, backScreen: MultiplayerScreen){
+        //Create a popup
+        val popup = Popup(this)
+        popup.addGoodSizedLabel("Working...".tr()).row()
+        popup.open()
+
+        thread {
+            try {
+                //download to work with newest game state
+                val gameInfo = OnlineMultiplayer().tryDownloadGame(gameId)
+                val playerCiv = gameInfo.currentPlayerCiv
+
+                //only give up if it's the users turn
+                //this ensures that no one can upload a newer game state while we try to give up
+                if (playerCiv.playerId == game.settings.userId){
+                    //call next turn first to make sure everything works like intended
+                    //this skips everything like trade request which will be handled by the AI on the following turn
+                    gameInfo.nextTurn()
+                    //Set own civ info to AI
+                    playerCiv.playerType = PlayerType.AI
+                    playerCiv.playerId = ""
+
+                    //Add notification so everyone knows what happened
+                    //call for every civ cause AI players are skipped anyway
+                    for (civ in gameInfo.civilizations){
+                        civ.addNotification("[${playerCiv.civName}] gave up and is now controlled by AI".tr(), Color.RED)
```

Comment:
DON'T .tr() this, because then you 'lock' the notification to your language. It should be left untranslated until displayed to the player, who can then get it in his language 🧠